BASE/Common.props

```diff
@@ -6,7 +6,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Directory.Build.props'))\Directory.Build.props" Condition="'$(EnlistmentRoot)' == ''"/>
 
   <PropertyGroup>
-    <LangVersion>7.1</LangVersion>
     <ChecksumAlgorithm>SHA256</ChecksumAlgorithm>
```

Comment:
I believe it's safe to remove this. SHA256 is now the default.

Can you also test if this file can be removed? :)